test/Microsoft.AspNetCore.Hosting.Tests/WebHostTests.cs

```diff
@@ -768,6 +768,9 @@ public Action<IApplicationBuilder> Configure(Action<IApplicationBuilder> next)
         [Fact]
         public void EnvDefaultsToProductionIfNoConfig()
```

Comment:
We should delete this test and write a functional or end to end test that does this.